pkgs/applications/networking/instant-messengers/webcord/webcord-vencord/vencord.nix

```diff
@@ -0,0 +1,23 @@
+{
+  runCommand,
+  lib,
+  curl,
+  p7zip,
+}:
+# Fetchzip doesn't recognize the crx file as zip.
+runCommand "vencord-extension" {
+  version = "1.1.5";
+  meta.license = lib.licenses.gpl3Only;
+  VENCORD_CRX_URL = "https://clients2.google.com/service/update2/crx?response=redirect&acceptformat=crx2,crx3&prodversion=109.0&x=id%3Dcbghhgpcnddeihccjmnadmkaejncjndb%26installsource%3Dondemand%26uc";
+  outputHash = "sha256-lovKwzBa4IMKDZ4TR6lv9pzWV7Vjm1FXG8ZRv4dZ9IM=";
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  nativeBuildInputs = [curl p7zip];
+  downloadToTemp = false;
+} ''
```

Comment:
I don't think we want to hack together our own fetcher here

Reply:
not using the prebuild anymore